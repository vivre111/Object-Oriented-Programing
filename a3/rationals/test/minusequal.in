create a 1/2 create b 0/1 create c -3/4 create d 6/2
-= a a print a -= a b print a -= a c print c - c a print c - c a print a
