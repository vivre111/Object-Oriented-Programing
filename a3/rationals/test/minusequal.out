0
0
0
0
3/4
-3/4
-3/2
-3/4
-3/2
3/4
